test/behavior/nesting.ts

```diff
@@ -544,15 +586,17 @@ async function shouldBehaveLikeNesting(
       const toOwner = tokenOwner.address;
       const notOwner = addrs[3];
       await expect(
-        parent.connect(notOwner).unnestChild(parentId, 0, toOwner, false),
+        parent.connect(notOwner).unnestChild(parentId, toOwner, 0, child.address, childId, false),
       ).to.be.revertedWithCustomError(child, 'ERC721NotApprovedOrOwner');
     });
 
-    it('cannot unnest not existing child', async function () {
+    it('cannot unnest from not existing parent', async function () {
```

Comment:
Can we add the correct example to check that a non-existent child can't be unnested as well?
It might be hidden by the GitHub UI, so it might already exist

Reply:
The non existent child is now covered on `cannot accept child if address or id do not match` and `cannot accept children for non existing index` 
This was just a bad name from the beginning

Reply:
Great, then I see no additions needed 👌